wpilib/src/notifier.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 First Rust Competition Developers.
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// use std::time::Duration;
+use wpilib_sys::*;
+
+#[derive(Debug)]
+/// An FPGA notifier alarm.
+pub struct Alarm {
```

Comment:
Alarms should probably keep the original name `Notifier` from wpilib.

Reply:
A WPILib Notifier is a thread that calls a function periodically. Since this _isn't_ what the WPILib Notifier does, I'm not calling this `Notifier`, since that'd be misleading.

Reply:
Also note the commented out code at the bottom of this module where I started to write an actual Notifier implementation (but gave up not-even-halfway).